code/controllers/subsystem/abnormality_queue.dm

```diff
@@ -1,7 +1,7 @@
 SUBSYSTEM_DEF(abnormality_queue)
 	name = "Abnormality Queue"
 	flags = SS_KEEP_TIMING | SS_BACKGROUND
-	wait = 5 MINUTES
+	wait = INFINITY
```

Comment:
If you are going to do this, then just make it not fire at all.

Reply:
Yeah, temp fix that I'll change.